argtable3/LICENSE

```diff
@@ -0,0 +1 @@
+argtable3/LICENSE
```

Comment:
likewise, this is not a valid LICENSE file. We can copy the upstream's

Reply:
`argtable3/LICENSE` is a symbolic link to `argtable3/argtable3/LICENSE` exactly like the `expat/LICENSE` is a symbolic link to `expat/expat/README.md` (see https://github.com/espressif/idf-extra-components/blob/master/expat/LICENSE).

I assumed that I could reuse this strategy in argtable3 since it is used in other components of the repo. Could you let me know why I can't use the symbolic link approach?